.travis.yml

```diff
@@ -61,6 +61,9 @@ matrix:
 before_install:
   # Exit build early if only documentation was changed in a Pull Request.
   - source ${BLT_DIR}/scripts/travis/exit_early
+  # Prepare local testing version of blt-project so it can be used by ORCA.
+  - cp -r subtree-splits/blt-project /tmp/
+  - composer config --global repositories.blt path /tmp/blt-project
```

Comment:
That is _beautiful_ simplicity. I can't wait to see how this works.

Reply:
I know, right? I was shocked that this worked on the first try!